### ISSUE TITLE
calling toFloat with a number results in TypeError

### ISSUE THREAD

**Issue by roosephu** [RocksonZeta/koa-validate#22]
When calling `isFloat`, the library checks whether the object is already a number. But when calling `toFloat` the library doesn't check that, so passing a number to `v.toFloat` will result in `TypeError: This library (validator.js) validates strings only`.

It may not be a big problem, since we should know the type of each object before validating.

**Comment by RocksonZeta** [comment]
which version of your koa-validate? please try latest version .

**Comment by roosephu** [comment]
I'm now using koa-validate v1.0.3. 

Here are some code I see in this repo, from Line 502 to 508:

```javascript
Validator.prototype.toFloat = function(tip) {
	this.isFloat(tip);
	if (!this.hasError()) {
		this.value = this.params[this.key] = v.toFloat(this.value);
	}
	return this;
};
```

If `this.value` is a number, it'll throw an error.

**Comment by RocksonZeta** [comment]
thanks, this bug has been fixed at 1.0.4
